### PR TITLE
BIGTOP-4153: Enhance Bigtop package.gradle: Add pkgSuffix and parentD…

### PR DESCRIPTION
**Pull request by JiaLiangC**
…ir Build Arguments for Deb Package

### Description of PR
Update package.gradle to pass pkgSuffix and parentDir parameters to the Debian package scripts to support the "Add pkgSuffix and parentDir Build Arguments for Deb Package" feature.

### How was this patch tested?
1.test with   pkgSuffix and parentDir parameters 
image

2.test without   pkgSuffix and parentDir parameters 
image

3.run zk smoke test on ubuntu-22
image

### For code changes:

- [ ] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'BIGTOP-3638. Your PR title ...')?
- [ ] Make sure that newly added files do not have any licensing issues. When in doubt refer to https://www.apache.org/licenses/

### CODE REVIEW

File: packages.gradle [hunk 1]
```diff
@@ -511,7 +521,10 @@ def genTasks = { target ->
     // Creating source package
     exec {
       workingDir DEB_BLD_DIR
+      environment 'PARENT_DIR', FULL_PARENT_DIR
+      environment 'PKG_NAME_SUFFIX', DEB_PKG_NAME_SUFFIX
       commandLine "dpkg-buildpackage -uc -us -sa -S".split(' ')
+      environment 'bigtop_base_version', BIGTOP_BASE_VERSION
```
Comment (iwasakims) [inline packages.gradle:527]: formatting nits: the line 527 should be before 526?

File: packages.gradle [hunk 2]
```diff
@@ -382,7 +382,10 @@ def genTasks = { target ->
     def final BASE_VERSION = config.bigtop.components[target].version.base
     def final SRCDEB = "${PKG_NAME}_$PKG_VERSION-${BIGTOP_BUILD_STAMP}.dsc"
     def final HADOOP_VERSION = config.bigtop.components["hadoop"].version.pkg
+    def final FULL_PARENT_DIR = getParentDir(config.bigtop.base_version, "deb")
+    def final DEB_PKG_NAME_SUFFIX =  getPkgNameSuffix(config.bigtop.components[target], "deb").pkgNameSuffix
 
+    def final BIGTOP_BASE_VERSION = "${config.bigtop.base_version}"
```
Comment (iwasakims) [inline packages.gradle:388]: looks like unintentionally inserted after blank line instead of before.

**Comment by JiaLiangC** [comment]
@iwasakims Could you help check again?

**Review by iwasakims — APPROVED** [review]
LGTM. Thanks, @JiaLiangC.

**Pull request merged**